tests/ecmp/test_fgnhg.py

```diff
@@ -172,8 +176,31 @@ def partial_ptf_runner(ptfhost, test_case, dst_ip, exp_flow_count, **kwargs):
             qlen=1000,
             log_file=log_file)
 
+def reprogram_ecmp_routes(duthost, prefix_list, ip_to_port, ipcmd):
+    logger.info ("Reprogramming ECMP routes")
+    vtysh_base_cmd = "vtysh -c 'configure terminal'"
+    t_end = time.time() + 60 * 6
+    while time.time() < t_end:
+        for prefix in prefix_list:
+           cmd = vtysh_base_cmd
+           for nexthop in ip_to_port:
+               cmd = cmd + " -c '{} {} {}'".format(ipcmd, prefix, nexthop)
+           configure_dut(duthost, cmd)
```

Comment:
warm reboot will introduce control plane down time. How could this continuous reprogramming work?